### PR TITLE
update ids for existing pingdom checks

### PR DESCRIPTION
**Pull request by ilaflamme-va**

### CODE REVIEW

File: pingdom-checks/appeals.ping [hunk 1]
```diff
@@ -31,5 +31,5 @@ pingdom save-check \
   -a url="/services/appeals/v0/healthcheck" \
   -a apikey="$SANDBOX_APPEALS_API_KEY" \
   -a group="appeals" \
-  -a userids_csv="$NO_USERS" \
+  -a userids_csv="$STATUSPAGE_SANDBOX_APPEALS_STATUS" \
```
Comment (jhulbert-va) [inline pingdom-checks/appeals.ping:34]: Do these dev checks need to be sent to statuspage? We're just going to remove them within the next sprint or 2 no?
In addition, this being sent to the sandbox status page address just doubles up on health checks being sent right?

Comment (ilaflamme-va) [reply pingdom-checks/appeals.ping:34]: The scope of the story was to update the IDs, so I'm not sure.

Comment (bschofield-va) [reply pingdom-checks/appeals.ping:34]: The scope of the story is 
> Integrate Pingdom and Statuspage for this API in Staging and Production environments

File: pingdom-checks/appeals.ping [hunk 2]
```diff
@@ -21,15 +21,5 @@ pingdom save-check \
   -a url="/services/appeals/v0/healthcheck" \
   -a apikey="$SANDBOX_APPEALS_API_KEY" \
   -a group="appeals" \
-  -a userids_csv="$NO_USERS" \
-  -a integrationids_csv="$HEALTH_APIS_SLACK_ID"
-
-pingdom save-check \
-  --template request-with-apikey \
-  -a name=dev-appeals-v0-health-check \
```
Comment (jhulbert-va) [inline pingdom-checks/appeals.ping:29]: I dont think we want these removed, just not integrated with statuspage.

Comment (eclendenning-va) [reply pingdom-checks/appeals.ping:29]: Correct, not removing them just  yet, that is part of SHANK-550 which is on hold until given the all clear to remove them by devops.

Comment (bschofield-va) [reply pingdom-checks/appeals.ping:29]: for clarity ... we're not going to remove the checks against `dev-api.va.gov`. Those will remain in Pingdom. We will remove the Development environment from Statuspage. Dev-api is now just for us so the general public does not need to know about it.

**Pull request merged**